test/unit/module/algo/algorithm/min_element_special_cases.cpp

```diff
@@ -11,11 +11,11 @@
 
 #include <span>
 
-TTS_CASE("Min element one pass, uint8 index")
+TTS_CASE("Min element one pass, uint16 index")
 {
   auto alg = eve::algo::min_element         //
       [eve::algo::single_pass]              //
-      [eve::algo::index_type<std::uint8_t>] //
```

Comment:
No - the point was specifically to tests index uint8.

Reply:
Ok, will replace back. There was a problem with rvv with `vlen=128`:

We can process 128 elements by one operation  and with `unroll_count==2` we get overflow for uint8_t.

Will forbid this test in next iteration.

Reply:
Done